indico/modules/events/timetable/templates/balloons/_balloon_utils.html

```diff
@@ -33,3 +33,19 @@
         {% endif %}
     </dl>
 {% endmacro %}
+
+{% macro render_palette_picker(obj, endpoint, color_list) %}
+    <dl class="i-data-list thirty-seventy">
+        <dt>{% trans %}Color{% endtrans %}</dt>
+        <dd>
+            <a class="i-button icon-palette palette-picker palette-picker-trigger"
+               style="{{ obj.colors.css }}"
+               data-method="PATCH"
+               data-href="{{ url_for(endpoint, obj) }}"
+               {# forceescape needed to properly handle the double quotes inside |tojson's output #}
```

Comment:
a bit verbose? you always need to `|forceescape` when using `tojson` outside `<script>` tags and in all the other places we don't have such a comment either